core/src/banking_stage.rs

```diff
@@ -338,6 +338,12 @@ pub struct BatchedTransactionCostDetails {
     pub batched_write_lock_cost: u64,
     pub batched_data_bytes_cost: u64,
     pub batched_execute_cost: u64,
+
+    pub batched_retried_txs_per_block_limit_count: u64,
+    pub batched_retried_txs_per_vote_limit_count: u64,
+    pub batched_retried_txs_per_account_limit_count: u64,
+    pub batched_retried_txs_per_account_data_block_limit_count: u64,
+    pub batched_dropped_txs_per_account_data_total_limit_count: u64,
```

Comment:
these can be pulled into a separate `BatchedTransactionErrors` as well

Reply:
Done!